flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricRegistryImpl.java

```diff
@@ -87,10 +86,14 @@
 
 	private boolean isShutdown;
 
+	public MetricRegistryImpl(MetricRegistryConfiguration config) {
+		this(config, Collections.emptyList());
+	}
+
 	/**
 	 * Creates a new MetricRegistry and starts the configured reporter.
 	 */
-	public MetricRegistryImpl(MetricRegistryConfiguration config) {
+	public MetricRegistryImpl(MetricRegistryConfiguration config, List<ReporterSetup> reporterConfigurations) {
```

Comment:
Could this be a `Collection<ReporterSetup>`?

Reply:
yes